examples/shell.emscripten.html

```diff
@@ -1,3 +1,17 @@
+// Copyright 2023 The Marl Authors.
```

Comment:
HTML quote is `<!-- comment -->`, but some browsers apparently can't handle a comment before the `<!doctype html>`.
I'd recommend just adding this file to the [`license-checker.cfg` `"exclude"` list](https://github.com/google/marl/blob/9c689c9a88bd8c6a8fe48b01d5ec6b71974282a6/license-checker.cfg#L8-L22):

Reply:
exemption added